bin/prod/update.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+if ps aux| grep -v grep |grep --quiet rclone; then
+  echo "rclone is running, not redeploying at this time"
+  echo "in future, consider adding delays/retries here"
+  exit 1
+else
+  echo rclone is not running, ok to deploy
+fi
+
+
+echo updating code...
+sudo bash -c "(cd /root/motuz && git pull)" # # TODO unhardcode motuz dir
+# TODO - make sure we are in the right branch?
+echo running redeploy script...
+# TODO - disable 'sleep' in redeploy script if calling from here, to minimize downtime
+sudo bash -c "(cd /root/motuz && bin/prod/redeploy.sh)"
```

Comment:
Sorry, some refactoring happened. `redeploy.sh` is no longer inside `bin/prod`. Actually that folder does not exist anymore. Can we make a `ci` folder and put this file (`bin/ci/update.sh`) there instead? Then change this line to refer to the `redeploy.sh` that moved 1 level up?

Reply:
Just did that in de0126b674835585d3c1bcb13dbf4b160fb65068 .